frontend/views/containers/sidebar/GroupsList.vue

```diff
@@ -81,42 +53,82 @@ export default {
 <style lang="scss" scoped>
 @import "../../../assets/style/_variables.scss";
 
-.c-item {
-  margin-bottom: 0.125rem;
+.c-group-list {
+  width: 3.5rem;
+  padding-top: 0.7rem;
+  background-color: $general_1;
 }
 
-.c-trigger {
-  padding: 0 $spacer-sm;
-  margin: 0 $spacer-sm $spacer-sm $spacer-sm;
-  width: calc(100% - 1rem);
-  line-height: 1;
-  overflow: hidden;
-  background-color: $primary_2;
-  transition: background-color ease-in 0.3s;
-  justify-content: flex-start;
-  font-size: $size-4;
-  font-weight: bold;
+.c-group-list-item {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 3.2rem;
+  margin-bottom: 0.25rem;
+}
 
-  .c-avatar {
-    margin-right: 0.5rem;
-    transition: transform cubic-bezier(0.18, 0.89, 0.32, 1.28) 0.3s;
+.group-badge {
+  &::before {
+    content: "";
+    position: absolute;
+    width: 3rem;
+    height: 3rem;
+    border: 1px solid transparent;
+    border-radius: 50%;
+    transform: scale(0.7);
+    transition: all .25s cubic-bezier(0.18, 0.89, 0.32, 1.38);
   }
 
-  i {
-    color: inherit;
-    margin-left: auto;
-    margin-right: 0.5rem;
+  &.is-active {
+    &::before {
+      transform: scale(1);
+      border-color: $text_0;
+      animation: spin 0.5s ease-out;
+    }
   }
 
-  &:hover,
-  &:focus {
-    box-shadow: none;
+  &:not(.is-active) {
+    cursor: pointer;
+    &:hover::before {
```

Comment:
Don't forget `:focus` state as part of possible user interactions. (Please review the other places too where there's only `:hover`)